src/components/LegislativeProcess/ProcessCard.svelte

```diff
@@ -0,0 +1,12 @@
+<script lang="ts">
+	export let imgSrc: string;
+	export let title: string;
+</script>
+
+<div class="flex flex-col items-center bg-green-10 p-5">
+	<img src={imgSrc} alt="process-step-${title}" />
```

Comment:
Svelte doesn't use `$` in front of the variable bracket when used in an attribute. In this case, if the title = "ชื่อ", the alt will be "process-step-$ชื่อ" which I'm not sure if it is what you want.

Other things looking good now. I already reach out to the designer and they should fix the image problem. I'll let you know when it's ready so you can export the correct images.